io.github.Qalculate.qalculate-qt.yaml

```diff
@@ -0,0 +1,86 @@
+app-id: io.github.Qalculate.qalculate-qt
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: qalculate-qt
+rename-appdata-file: qalculate-qt.appdata.xml
+rename-desktop-file: qalculate-qt.desktop
+rename-icon: qalculate-qt
+copy-icon: true
+finish-args:
+  - --device=dri
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # Network access
+  # for Update Exchange Rates
+  - --share=network
+  # Filesystem access
+  # Import CSV File
+  # Export CSV File
+  # Save Result Image
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-public-share
```

Comment:
Since it's a Qt app, it should seamlessly open/save files through portals. Does something not work properly without direct filesystem access?

Reply:
Those lines were copied from the flatpak for the GTK GUI (io.github.Qalculate). I tested without those lines and, except for some minor functionality (user import/export of CSV file using mathematical function instead of GUI dialog), it seems to work properly.

Reply:
The GTK UI app doesn't seem to use portals (probably calls `FileChooser` instead of `FileChooserNative`?).

Reply:
I have not removed direct file system access. I will attempt to use FileChooserNative in the GTK UI.